### ISSUE TITLE
piping crash in 6.16.1

### ISSUE THREAD

**Issue by furiozo-ga** [kartikk221/hyper-express#267]
version 6.16.1 introduces a problem related to stream piping.
the following code works fine with 6.16.0 but crashes 6.16.1
```
import HyperExpress from 'hyper-express'
import http from 'node:http'
const web = new HyperExpress.Server()

web.get('/test',(request, response) => response.send('test'))

web.get('/', (request, response) => {
   http.get(`http://localhost:1234/test`, (readable) => {
      readable.pipe(response)
   })
})

web.listen(1234)
```

`curl localhost:1234`

```
node:internal/streams/writable:474
      throw new ERR_INVALID_ARG_TYPE(
      ^

TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined
    at _write (node:internal/streams/writable:474:13)
    at Writable.write (node:internal/streams/writable:502:10)
    at Response.send (/home/furi/js/node_modules/hyper-express/src/components/http/Response.js:493:32)
    at Response.end (/home/furi/js/node_modules/hyper-express/src/components/compatibility/ExpressResponse.js:50:21)
    at Response.passthrough [as end] (/home/furi/js/node_modules/hyper-express/src/components/http/Response.js:961:29)
    at IncomingMessage.onend (node:internal/streams/readable:946:10)
    at Object.onceWrapper (node:events:633:28)
    at IncomingMessage.emit (node:events:531:35)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'ERR_INVALID_ARG_TYPE'
}

Node.js v20.14.0
```

**Comment by kartikk221** [comment]
Hey, I think the line which is causing the bug is likely https://github.com/kartikk221/hyper-express/blob/45e0e9f353dc72b1e379e51dedeccabb97e15641/src/components/http/Response.js#L493 could you try going into your `node_modules` folder, tracking down `hyper-express` then opening the file `/src/components/http/Response.js` and the changing line 493 to:
```js
if (body) this._writable.write(body);
```

Let me know If that fixes your bug, If so I will push an update with it.

**Comment by furiozo-ga** [comment]
yes, this fixes the crash.

**Comment by kartikk221** [comment]
Hey, this fix has been pushed in the latest `v6.16.3` update.
